### ISSUE TITLE
Not work with google_mobile_ads 0.13.0

### ISSUE THREAD

**Issue by lyoko515** [hsangtini/flutter_admob_app_open#13]
I tried flutter_admob_app_open 1.0.4 with google_mobile_ads 0.13.0 on android and it didn't work

**Comment by phanvanluan94** [comment]
Same issue on android.

**Comment by youngminz** [comment]
I can confirm #11 PR works with latest version of google_mobile_ads. (currently 0.13.3 anyway) Try this:

```
  flutter_admob_app_open:
    git:
      url: https://github.com/mattcianna/flutter_admob_app_open
      ref: 8fc1caa9d64985da111545332003165e7d24f520
```

**Comment by hsangtini** [comment]
Fixed at last version 1.1.1
